app/views/evaluators/index.html.erb

```diff
@@ -64,9 +68,9 @@
         <%= inline_error(form, :email) %>        
       </div>
 
-      <%= form.button type: 'submit', class: "usa-button margin-top-5 margin-bottom-3" do %>
-        <%= image_tag('images/usa-icons/person.svg', class: "usa-icon--size-3 icon-white", alt: "") %>
-        <span>Add to evaluator list</span>
+      <%= form.button type: 'submit', class: "usa-button margin-top-5 margin-bottom-3 padding-2" do %>
+        <%= image_tag('images/usa-icons/check.svg', class: "usa-icon icon-white", alt: "save") %>
+        Add Evaluator
       <% end %>
```

Comment:
Figma and #388 added a Cancel button next to Add Evaluator, which clears the form fields:
<img width="265" alt="Screenshot 2025-03-06 at 11 28 46 AM" src="https://github.com/user-attachments/assets/8b192fd5-f505-44bf-bb25-a5959f39fdc4" />